fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -1352,6 +1362,18 @@ public static QueryParameter buildInclusionCriteria(String compartmentName, Set<
         return rootParameter;
     }
 
+    /**
+     * Check the configuration to see if the flag enabling the compartment search
+     * optimization. Defaults to false so the behavior won't change unless it
```

Comment:
I think this comment is old...defaults to true now

Reply:
yeah, the only change I made is moving this method down (so that the parseCompartmentQueryParameters overloads would be next to each other).  but you're right!  will fix now

Reply:
done